app/src/main/res/layout/all_episodes_fragment.xml

```diff
@@ -1,15 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-             xmlns:tools="http://schemas.android.com/tools"
-             android:orientation="vertical"
-             android:layout_width="match_parent"
-             android:layout_height="match_parent">
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/txtvInformation"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:paddingTop="2dp"
+        android:paddingBottom="2dp"
+        android:visibility="gone"
+        tools:text="(i) Information" />
 
     <android.support.v7.widget.RecyclerView
         android:id="@android:id/list"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_below="@+id/txtvInformation"
+        android:layout_above="@+id/progLoading"
```

Comment:
Please remove this line

Reply:
I actually never see the spinner, unless I comment out hiding it after loading data.
Is it only visible when no episodes are visible?

Reply:
If I leave out the first of the two lines, the filter text indicator will overlap the first episode listed.

Reply:
> I actually never see the spinner, unless I comment out hiding it after loading data.

You just need to get a slow phone :D I see that spinner every time I click on a sidebar entry.